### PR TITLE
[#230] Update RIM details page to display PCRs

### LINKED ISSUE #230: Add upload of Support RIM and display expected PCRs on RIM Details page
Issue #212 added a RIM details page.
Issue #222 added the ability to process a TCG Event and produce a set of Expected TPM PCRs.

What's needed is an ability to upload the Support RIM and see the Expected PCR Values on the RIM details page. The support RIM can of 2 types: Event Log Assertions and TPM PCR Assertions.

Event Log file names are expected to have a .rimel file name extension for Event Log Assertions.
Event Log file names are expected to have a .rimpcr  file name extension for TPM PCR Assertions.

This issue only adds support for the Event log assertions.

### PR DESCRIPTION
**Pull request by cyrus-dev**
The main update was to the details page controller for the RIM objects.  It uses the TCGLogProcessor class to pull in the information.  The initial update caused problems (with no error information) when the file associated with the RIM file didn't exist.

Closes #230

### CODE REVIEW

File: HIRS_Utils/src/main/java/hirs/tpm/eventlog/CryptoAgileEventLog.java [hunk 1]
```diff
@@ -1,138 +1,11 @@
 package hirs.tpm.eventlog;
 
-import java.io.ByteArrayInputStream;
```
Comment (chubtub) [inline HIRS_Utils/src/main/java/hirs/tpm/eventlog/CryptoAgileEventLog.java]: Do you need to keep this class if it's been taken over by TpmPcrEvent2?

File: HIRS_Utils/src/main/java/hirs/tpm/eventlog/SHA1EventLog.java [hunk 2]
```diff
@@ -1,134 +1,11 @@
 package hirs.tpm.eventlog;
 
-import java.io.ByteArrayInputStream;
```
Comment (chubtub) [inline HIRS_Utils/src/main/java/hirs/tpm/eventlog/SHA1EventLog.java]: Do you need to keep this class if it's been taken over by TpmPcrEvent1?

File: HIRS_AttestationCAPortal/src/main/java/hirs/attestationca/portal/page/controllers/ReferenceManifestPageController.java [hunk 3]
```diff
@@ -177,17 +187,44 @@ protected RedirectView upload(
...
         // loop through the files
         for (MultipartFile file : files) {
+            fileName = file.getOriginalFilename();
+            if (fileName.toLowerCase().endsWith("swidtag")) {
+                rims.add(file);
+            } else {
+                filePath = Paths.get(String.format("%s/%s",
+                            SwidResource.RESOURCE_UPLOAD_FOLDER,
+                            file.getOriginalFilename()));
+                if (Files.notExists(Paths.get(SwidResource.RESOURCE_UPLOAD_FOLDER))) {
```
Comment (chubtub) [inline HIRS_AttestationCAPortal/src/main/java/hirs/attestationca/portal/page/controllers/ReferenceManifestPageController.java]: Should this check occur before the Paths.get(String....) call immediately preceding it?

Comment (cyrus-dev) [reply HIRS_AttestationCAPortal/src/main/java/hirs/attestationca/portal/page/controllers/ReferenceManifestPageController.java]: The Path variable isn't acting on the location provided, so if it doesn't exists, nothing bad happens.

**Pull request merged**